src/runtime/arc.ts

```diff
@@ -73,7 +74,7 @@ type DeserializeArcOptions = Readonly<{
 }>;
 
 @SystemTrace
-export class Arc implements ArcInterface {
+export class Arc implements ArcInterface, SearchableStore {
```

Comment:
It seems like an ArcInterface should extend StoreContext

Reply:
Also likely `Manifest` should implement `StoreContext`.

Reply:
Nice, I like that.